pkg/blob/controllerserver.go

```diff
@@ -442,7 +442,8 @@ func (d *Driver) ValidateVolumeCapabilities(ctx context.Context, req *csi.Valida
 		if resourceGroupName == "" {
 			resourceGroupName = d.cloud.ResourceGroup
 		}
-		blobContainer, err := d.cloud.BlobClient.GetContainer(ctx, resourceGroupName, accountName, containerName)
+		blobContainer, retryErr := d.cloud.BlobClient.GetContainer(ctx, subsID, resourceGroupName, accountName, containerName)
+		err = retryErr.Error()
```

Comment:
`retryErr` could be nil, there retryErr.Error() would panic

Reply:
this scenario is handled in [Error()](https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/vendor/sigs.k8s.io/cloud-provider-azure/pkg/retry/azure_error.go#L77-L79) , that is when retryErr is nil, retryErr.Error() will be nil too.